test/runtests.jl

```diff
@@ -39,7 +39,7 @@ featurecollections = [g, multipolygon, realmultipolygon, polyline, point, pointn
 
         f1, _ = iterate(t)
         @test f1 isa GeoJSONTables.Feature
-        @test isempty(Base.propertynames(f1)) # .== [:cartodb_id, :addr1, :addr2, :park])
+        @test !isempty(Base.propertynames(f1)  .== [:cartodb_id, :addr1, :addr2, :park])
```

Comment:
This doesn't seem right. You are checking if the comparison is empty.

Could you add some more tests? Now we only know it's not empty, but not that it is working. You could check the keys, values, types etc.

Reply:
Oh yeah sure. I had planned on adding more tests later on, for now I wanted to align to the tests you had added.

Reply:
We probably redo the testing at some point yeah, but in general I think each new functionality should include at least some minimal tests, others you can assume in many cases that it doesn't work. At least that's how it works for me, most code I write and not test is broken.